Ports/openrct2/package.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=openrct2
+version=0.4.3
+auth_type=sha256
+depends=(
+    "SDL2"
+    "libicu"
+    "libpng"
+    "zlib"
+    "xz"
+    "libzip"
+    "freetype"
+    "fontconfig"
+    "libvorbis" "libogg" "speexdsp" "flac"
+    "openssl" "curl") # only for networking support
```

Comment:
This looks like a big list of dependencies. Are you sure that all of these are used directly, and not through some other dependency?

Reply:
Yes, with the exception of `xz` (I don't even remember why I added it). Most of them could be disabled, but since we already have them I decided to just enable them for a more complete build.